client/scripts/views/components/sidebar/tag_selector.ts

```diff
@@ -51,6 +96,7 @@ const TagRow: m.Component<ITagRowAttrs, {}> = {
       ],
       contentRight: [
         !hideEditButton && m('.tag-count', pluralize(count, 'post')),
+        !hideEditButton && m(FeaturedCheckbox, { ...vnode.attrs }),
```

Comment:
IMO, we should avoid using splats unless it is completely obviously clear what is being expanded. And we should avoid reusing IAttrs declarations between different components.